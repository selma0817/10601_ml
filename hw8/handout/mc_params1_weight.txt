-2.625282100999999813e+00 1.376415137362733798e+00 -1.189262535543432475e-03
-2.615242606000000247e+00 1.372704718250047584e+00 5.422881458320114338e-04
-2.623842649000002858e+00 1.378291718460027626e+00 1.439101096681692522e-03
