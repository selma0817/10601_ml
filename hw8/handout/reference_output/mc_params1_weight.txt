-2.625282100999999813e+00 1.376415137362734020e+00 -1.189262535543432042e-03
-2.615242606000000247e+00 1.372704718250047584e+00 5.422881458320121927e-04
-2.623842649000002858e+00 1.378291718460027848e+00 1.439101096681694040e-03
